modules/apps/frontend-js/frontend-js-spa-web/src/main/java/com/liferay/frontend/js/spa/web/internal/servlet/taglib/helper/SPAHelper.java

```diff
@@ -232,9 +239,32 @@ private long _getCacheExpirationTime(SPAConfiguration spaConfiguration) {
 		return cacheExpirationTime;
 	}
 
-	private static final String _REDIRECT_PARAM_NAME;
+	private JSONArray _getExcludedPathsJSONArray(
+		SPAConfiguration spaConfiguration) {
+
+		JSONArray jsonArray = JSONFactoryUtil.createJSONArray();
+
+		String[] excludedPaths = StringUtil.split(
+			SPAConfigurationUtil.get("spa.excluded.paths"));
```

Comment:
Hey @robertoDiaz, do you see any specific value in getting that stored in its own `spa.properties` file?

Seems like a lot of unnecessary indirection. If we just add our defaults directly here then we can get rid of:
- `spa.properties`
- `SPAConfigurationUtil`

Reply:
yes, doing this we do not allow users to remove them (this could drive into errors if removed)

Reply:
I'm not saying add them as configurable paths, simply do:

```java
JSONArray jsonArray = JSONFactoryUtil.createJSONArray();

jsonArray.put(_portal.getPathContext() + "/c/document_library");
jsonArray.put(_portal.getPathContext() + "/documents");
jsonArray.put(_portal.getPathContext() + "/image");

String[] customExcludedPaths = spaConfiguration.customExcludedPaths();

...

```

Which should be the same but we don't need to fetch the list of paths from a file and customers still can't remove them causing errors unintentionally.

Reply:
Of course we could (in fact we could had done this before, because the current spa.properties in very close to be a container for hardcoded variables).

I didn't do so in the PR because this "cosmetic" changes are out of my scope :D

If you prefer it this way, no problem at all, I could remove this file and the util class in a blink :D (I could make it look like an accident if you want ;D)

Reply:
@robertoDiaz Chema's out today so I would say just go ahead and do it so that we don't have to delay another day waiting for his confirmation.

Reply:
hey @wincent 

Thank you!!

I'll update the PR with this changes :)